woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -252,6 +252,13 @@ def test_unknown_inference_all_null(nulls):
             assert isinstance(inferred_type, Unknown)
 
 
+def test_unknown_inference_empty_series(empty_series):
+    for series in empty_series:
+        inferred_type = ww.type_system.infer_logical_type(series.astype("string"))
+        inferred_type.transform(series)
```

Comment:
I don't think this line is doing anything - you are just throwing away the return value from the `.transform()` call

Reply:
Will do! I wasn't sure if there were any important side effects to test here. I followed the pattern in `test_unknown_inference_all_null`, which called this function.

Reply:
I guess this *could* be testing that it is possible to convert the original series from the "object" type to the "string" type without an error.

Reply:
If that is the intent, I would maybe suggest we add a comment to that effect since it isn't immediately clear why what call would exist otherwise.